README.md

```diff
@@ -506,6 +512,119 @@ $ mbedgt -V --target=frdm-k64f-gcc
 
 Above command will execute all tests for yotta module you are in, e.g. ```mbed-drivers```.
 
+# Test specification JSON formatted input
+
+Greentea originally only supports yotta artefacts. It assumes it is run inside a yotta module and gathers information from local file system. To make it generic for any other test artefacts we can support a test specification input. This specification can tell information like platform, tool chain, build artefacts path, test binaries, flash methods of test binaries to Greentea.
```

Comment:
Nit-pick time!

> tool chain

to

> toolchain

(to be consistent with other instances of this word throughout the doc)